README.md

```diff
@@ -1,24 +1,179 @@
 
-* Configuration
+- Ruby
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li>RSPEC</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
 
-* How to run the test suite
+<!-- Features -->
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- Use database migration files to maintain database schema.
+- Understand ORM and database migrations and their implementation.
+- Processing data in models.
+- Views.
+- Forms
+- Validations and Model specs.
+- Integration specs for Views.
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 🚀 Live Demo <a name="live-demo"></a>
+
+Please click <a href="https://ror-budget-maker.onrender.com/" target="_blank">here</a> to go live with the application.
+
+### 🔭 Video Presentation <a name="video"></a>
+
+Please click <a href="https://youtu.be/GuNyhHLVo-c" target="_blank">here</a> to watch video presentation of the project
+
+<!-- GETTING STARTED -->
+
+## :computer: Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. Ruby on Rails installed and running. To get more information, read the installation guide.
+2. PostgreSQL installed and running
+
+### Setup
+
+```
+git clone https://github.com/DilsherB/ror-budget-maker.git
+cd ror-budget-maker
+```
+
+### Install
+
+- bundle install
+
+### Usage
+
+**To run the project follow the following instruction:-**
+
+- rails server or rails s
+
+### Run tests
+
+- To run tests, run the following command:
+
+```
+rspec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+## 👥 Author : <a name="authors"></a>
+
+👤 **Dilsher Balouch**
+
+- GitHub: [@DilsherB](https://github.com/DilsherB)
+- Linkedin: [Dilsher Balouch](https://www.linkedin.com/in/dilsher-balouch/)
+- Twitter: [@\_brilliantMindz](https://twitter.com/_brilliantMindz)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+:telescope: Future Features <a name="future-features"></a>
+
+- Add API endpoints to list user posts and comments.
+- Create an API Endpoint to create a comment for a post.
+- API documentation.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+:handshake: Contributing <a name="contributing"></a>
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+:star:️ Show your support <a name="support"></a>
+If you like this project give a :star:️. ⭐️
```

Comment:
- [optional] please kindly show your contribution and show your support title on different lines. this will make your readme more proffesional.
Yours
![Screenshot_28](https://github.com/DilsherB/ror-budget-maker/assets/65298048/d424838d-19d0-4774-9e77-1271472adf68)
original
![Screenshot_29](https://github.com/DilsherB/ror-budget-maker/assets/65298048/2dcce7e9-8488-476d-9a45-499e30bfbf72)